examples/TestEm3/TestEm3.cuh

```diff
@@ -28,6 +30,7 @@ struct Track {
   double initialRange;
   double dynamicRangeFactor;
   double tlimitMin;
+  double fPEmxSec;
```

Comment:
I am not experienced in this domain, but even then, this looks like a very cryptic variable name.